node/src/components/consensus/protocols/highway/round_success_meter.rs

```diff
@@ -4,15 +4,25 @@ use datasize::DataSize;
 
 use crate::{
     components::consensus::{
-        highway_core::{finality_detector::FinalityDetector, State, Weight},
+        highway_core::{finality_detector::FinalityDetector, round_id, State, Weight},
         traits::Context,
     },
     types::Timestamp,
 };
 
+// The number of most recent rounds we will be keeping track of.
 const NUM_ROUNDS_TO_CONSIDER: usize = 40;
+// The maximum number of failures allowed among NUM_ROUNDS_TO_CONSIDER latest rounds, with which we
+// won't increase our round length. Exceeding this threshold will mean that we should slow down.
 const MAX_FAILED_ROUNDS: usize = 10;
-const ACCELERATION_PARAMETER: u64 = 1000;
+// We will try to accelerate (decrease our round exponent) every `ACCELERATION_PARAMETER` rounds if
+// we have few enough failures.
+const ACCELERATION_PARAMETER: u64 = 40;
+// The maximum number of failures with which we will attempt to accelerate (decrease the round
+// exponent).
+const MAX_FAILURES_FOR_ACCELERATION: usize = 3;
+// The FTT, as a percentage, which we will use for looking for a summit in order to determine a
+// proposal's finality.
```

Comment:
It says _as percentage_ but it's a `u64`. How will the percentage be represented?

Reply:
`THRESHOLD = 1` means 1%. I can clarify that more in the comment if needed.

Reply:
Ah, so `total_w * THRESHOLD / 100` is supposed to mean that we are looking for quorum of 51% validators? Can we extract 100 to constant and add the comment?

Reply:
I'm not sure if it translates to 51% quorum (maybe? I don't remember the exact relationship between FTT and the quorum off the top of my head). In any case, the finality detector needs the FTT passed in as a weight, so we have to calculate the percentage of the total weight of all the validators manually, and this is what that calculation does.